src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -18,7 +18,6 @@
         <service id="api_platform.operation_method_resolver" class="ApiPlatform\Core\Bridge\Symfony\Routing\OperationMethodResolver" public="false">
             <argument type="service" id="api_platform.router" />
             <argument type="service" id="api_platform.metadata.resource.metadata_factory" />
-            <deprecated>The "%service_id%" service is deprecated since API Platform 2.5.</deprecated>
```

Comment:
💡 I haven't found a way to produce an XML file that can be loaded with DependencyInjection 4.4 and 5.1 without errors:
* DI 5.1 triggers a deprecation if the attributes `package` and `version` are missing.
* DI 4.4 throws an exception, if those attributes are present.

This is why I'm deprecating the services inside the extension now.